alpine.dockerfile

```diff
@@ -34,11 +34,9 @@ LABEL maintainer="Microsoft" \
 
 ARG JP_VERSION="0.2.1"
 
-WORKDIR azure-cli
-COPY . /azure-cli
-RUN apk add --no-cache ca-certificates bash bash-completion libintl icu-libs libc6-compat jq openssh-keygen \
+RUN --mount=type=bind,target=/azure-cli,source=./,rw apk add --no-cache ca-certificates bash bash-completion libintl icu-libs libc6-compat jq openssh-keygen \
```

Comment:
`rw` allows write to mount. Otherwise wheel build fails:
```
#6 9.533 Building wheels for collected packages: azure-cli-core, azure-cli, azure-cli-telemetry
#6 9.535   Building wheel for azure-cli-core (setup.py): started
#6 9.865   Building wheel for azure-cli-core (setup.py): finished with status 'error'
#6 9.870   error: subprocess-exited-with-error
#6 9.870
#6 9.870   × python setup.py bdist_wheel did not run successfully.
#6 9.870   │ exit code: 1
#6 9.870   ╰─> [5 lines of output]
#6 9.870       running bdist_wheel
#6 9.870       running build
#6 9.870       running build_py
#6 9.870       creating build
#6 9.870       error: could not create 'build': Read-only file system
#6 9.870       [end of output]
```